src/vcpkg/postbuildlint.cpp

```diff
@@ -1012,10 +1025,20 @@ namespace vcpkg::PostBuildLint
         const auto debug_bin_dir = package_dir / "debug" / "bin";
         const auto release_bin_dir = package_dir / "bin";
 
+        const auto lib_filter = [&pre_build_info]() {
+            if (pre_build_info.cmake_system_name.empty() || pre_build_info.cmake_system_name == "Windows" ||
+                pre_build_info.cmake_system_name == "WindowsStore")
+            {
+                if (pre_build_info.cmake_system_name == "MinGW") return NotExtensionsCaseInsensitive{{".lib", ".a"}};
```

Comment:
This condition is never true -- system name here is exactly empty, "Windows", or "WindowsStore". Was this intended to be an `else if()`?